apps/common/models.py

```diff
@@ -93,16 +96,12 @@ def is_bigcities_page(self):
         A page is a Big Cities page if it is either the BigCitiesHomePage, or a
         descendant of the BigCitiesHomePage.
         """
-        # Note: the BigCitiesHomePage inherits from StaticPage, which inherits from
-        # HipBasePage, so we must check self.staticpage.bigcitieshomepage and
-        # ancestor.staticpage.bigcitieshomepage.
         if (
             hasattr(self, "staticpage")
             and hasattr(self.staticpage, "bigcitieshomepage")
```

Comment:
I think this clause can be simplified to `if hasattr(self, "bigcitieshomepage")` now.

Reply:
Makes sense; thanks